libs/barista-components/formatters/index.ts

```diff
@@ -37,6 +37,8 @@ export { formatRate } from './src/rate/rate-formatter';
 export * from './src/rate/rate';
 export * from './src/bits/bits-formatter';
 export * from './src/bits/bits';
+/** @deprecated Please use dtDuration instead */
+/** @breaking-change - Removed with version 7.0.0 */
```

Comment:
Do we even need to have this in the barrel file, I guess deprecating the class / pipe is perfectly fine.